docs/docs/fundamentals/installation.md

```diff
@@ -4,98 +4,129 @@ title: Installation
+In order to fully utilize the [touch events](/docs/gestures/touch-events/) you also need to use `react-native-reanimated` 2.3.0 or newer.
 
-Note that if you wish to use [`React.createRef()`](https://reactjs.org/docs/refs-and-the-dom.html) support for [interactions](/docs/gesture-handlers/interactions/) you need to use v16.3 of [React](https://reactjs.org/)
+Setting up `react-native-gesture-handler` is pretty straightforward:
 
-In order to fully utilize the [touch events](/docs/gestures/touch-events/) you also need to use `react-native-reanimated` 2.3.0-beta.4 or newer.
+### 1. Start with installing the package from npm:
 
-## [Expo](https://expo.dev)
+<Tabs groupId="package-managers">
+  <TabItem value="expo" label="EXPO" default>
 
-To use the version of react-native-gesture-handler that is compatible with your Expo project, run `npx expo install react-native-gesture-handler`.
+    npx expo install react-native-gesture-handler
 
-The Expo SDK incorporates the latest version of react-native-gesture-handler available at the time of each SDK release, so Expo apps might not always support all our latest features as soon as they are available.
+  </TabItem>
+  <TabItem value="npm" label="NPM">
 
-### Bare [React Native](https://reactnative.dev/)
+    npm install react-native-gesture-handler
 
-Since the library uses native support for handling gestures, it requires an extended installation to the norm. If you are starting a new project, initialize it with [`npx create-expo-app@latest`](https://docs.expo.dev/get-started/create-a-project/) since it comes pre-installed with react-native-gesture-handler.
+  </TabItem>
+  <TabItem value="yarn" label="YARN">
 
-## JS
+    yarn add react-native-gesture-handler
 
-First, install the library using `yarn`:
+  </TabItem>
+</Tabs>
 
-```bash
-yarn add react-native-gesture-handler
+### 2. Wrap your app with `GestureHandlerRootView` component
+
+```jsx
+import { GestureHandlerRootView } from 'react-native-gesture-handler`;
+
+export default function App() {
+  return (
+    <GestureHandlerRootView>
+      <ActuallApp />
+    </GestureHandlerRootView>
+  );
+}
 ```
 
-or with `npm` if you prefer:
+If you don't provide anything to the `styles` prop, it will default to `flex: 1`. If you want to customize the styling of the root view, don't forget to also include `flex: 1` in the custom style, otherwise your app won't render anything. Keep `GestureHandlerRootView` as close to the actual root of the app as possible. It's the entry point for all gestures and all gesture relations. The gestures won't be recognized outside of the root view, and relations only work between gestures mounted under the same root view.
+
+If you're unsure if one of your dependencies alredy renders `GestureHandlerRootView` on its own, don't worry and add one at the root anyway. In case of nested root views, Gesture Handler will only use the top-most one and ignore the nested ones.
```

Comment:
```suggestion
If you're unsure if one of your dependencies already renders `GestureHandlerRootView` on its own, don't worry and add one at the root anyway. In case of nested root views, Gesture Handler will only use the top-most one and ignore the nested ones.
```
typo